game-app/game-core/src/main/java/games/strategy/triplea/ui/BattlePanel.java

```diff
@@ -263,8 +263,10 @@ public void showBattle(
                   defendingWaitingToDieCopy,
                   BattlePanel.this.getMap(),
                   battleType);
-          battleWindow.setTitle(
-              attacker.getName() + " attacks " + defender.getName() + " in " + location.getName());
+          String battleStr = battleType.toDisplayText();
+          battleStr = battleStr.equals("Battle") ? "" : String.format("  (%s)", battleStr);
```

Comment:
```suggestion
          final String battleStr = battleType == BattleType.NORMAL ? "" : String.format("  (%s)", battleType.toDisplayText());
```

Reply:
I'm assuming there is a 100 char per line length limit ? 

I've installed Intellij but Checkstyle has a problem with suppressions.xml, any idea what the problem is ? 
https://github.com/triplea-game/triplea/issues/9279
 
When the round info is relevant, the goal would to update the battleWindow title each round before the dice are rolled. 
It seems this require non trivial code modification. 
I think it would be best to go ahead with the current PR, and revisit later when v2.6 is out.

Reply:
Ah yes, forgot about that.
When using IntelliJ using the default keybindings you can press the Ctrl Key twice and then enter the command `gradle spotlessJavaApply --parallel` to do the formatting automatically for you